### PR TITLE
Run-to-run stability tests

### PR DESCRIPTION
**Pull request by SmirnovEgorRu**

### CODE REVIEW

File: daal4py/sklearn/test/test_run_to_run_stability_tests.py [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+    clf.fit(X, Y)
+    res  = [clf.predict(X), clf.predict_proba(X), clf.support_, clf.support_vectors_, clf.n_support_, clf.dual_coef_, clf.coef_, clf.intercept_]
+    name = ["clf.predict(X)", "clf.predict_proba(X)", "clf.support_", "clf.support_vectors_", "clf.n_support_", "clf.dual_coef_", "clf.coef_", "clf.intercept_"]
+    return res, name
+
+def run_svc_rbf(X, Y):
+    clf = SVC(random_state=0, probability=True, kernel='rbf')
+    clf.fit(X, Y)
+    res  = [clf.predict(X), clf.predict_proba(X), clf.support_, clf.support_vectors_, clf.n_support_, clf.dual_coef_, clf.intercept_]
+    name = ["clf.predict(X)", "clf.predict_proba(X)", "clf.support_", "clf.support_vectors_", "clf.n_support_", "clf.dual_coef_", "clf.coef_", "clf.intercept_"]
```
Comment (PetrovKP) [inline daal4py/sklearn/test/test_run_to_run_stability_tests.py:120]:
```suggestion
    name = ["clf.predict(X)", "clf.predict_proba(X)", "clf.support_", "clf.support_vectors_", "clf.n_support_", "clf.dual_coef_", "clf.intercept_"]

```

File: daal4py/sklearn/test/test_run_to_run_stability_tests.py [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+    return res, name
+
+def run_svc_rbf_const_gamma(X, Y):
+    clf = SVC(random_state=0, probability=True, kernel='rbf', gamma=0.01)
+    clf.fit(X, Y)
+    res  = [clf.predict(X), clf.predict_proba(X), clf.support_, clf.support_vectors_, clf.n_support_, clf.dual_coef_, clf.intercept_]
+    name = ["clf.predict(X)", "clf.predict_proba(X)", "clf.support_", "clf.support_vectors_", "clf.n_support_", "clf.dual_coef_", "clf.coef_", "clf.intercept_"]
```
Comment (PetrovKP) [inline daal4py/sklearn/test/test_run_to_run_stability_tests.py:127]:
```suggestion
    name = ["clf.predict(X)", "clf.predict_proba(X)", "clf.support_", "clf.support_vectors_", "clf.n_support_", "clf.dual_coef_",  "clf.intercept_"]

```

File: daal4py/sklearn/test/test_run_to_run_stability_tests.py [hunk 3]
```diff
@@ -0,0 +1,335 @@
+#*******************************************************************************
+# Copyright 2014-2020 Intel Corporation
```
Comment (PetrovKP) [inline daal4py/sklearn/test/test_run_to_run_stability_tests.py:2]:
```suggestion
# Copyright 2020 Intel Corporation

```

File: daal4py/sklearn/test/test_run_to_run_stability_tests.py [hunk 4]
```diff
@@ -0,0 +1,335 @@
...
+def run_pca_full(X, Y):
+    clf = PCA(n_components=0.5, svd_solver="full", random_state=0)
+    clf.fit(X)
+    res  = [clf.transform(X), clf.get_covariance(), clf.get_precision(), clf.score_samples(X), clf.components_, clf.explained_variance_, clf.explained_variance_ratio_, clf.singular_values_, clf.mean_, clf.noise_variance_]
+    name = ["clf.transform(X)", "clf.get_covariance()", "clf.get_precision()", "clf.score_samples(X)", "clf.components_", "clf.explained_variance_", "clf.explained_variance_ratio_", "clf.singular_values_", "clf.mean_", "clf.noise_variance_"]
+    return res, name
+
+def run_pca_daal4py_corr(X, Y):
+    algo = d4p.pca(resultsToCompute="mean|variance|eigenvalue", isDeterministic=True, method="correlationDense")
+    result1 = algo.compute(X)
```
Comment (PetrovKP) [inline daal4py/sklearn/test/test_run_to_run_stability_tests.py:234]:
```suggestion
    result = algo.compute(X)

```

**Review by PetrovKP — APPROVED** [review]

**Review by Pahandrovich — APPROVED** [review]

**Comment by PetrovKP** [comment]
It would be nice to refactoring, preferably with the help of pytest framework. And it would be necessary to uncomment the linear regression, check and at least merge in this form.

**Comment by SmirnovEgorRu** [comment]
Let's convert this in pytest and add more algorithms in next commits.

**Pull request merged**